src/util-logopenfile.h

```diff
@@ -50,7 +50,8 @@ typedef struct SyslogSetup_ {
 
 typedef struct ThreadLogFileHashEntry {
     uint64_t thread_id;
-    int slot_number; /* slot identifier -- for plugins */
+    int slot_number;    /* slot identifier */
+    int suri_thread_id; /* suri thread id -- for plugins */
```

Comment:
this comment isn't really telling me what the id is: is it internal, does it relate to a OS thread/process ID?

Why is the type `int`? Can it be negative?

Reply:
I'll update the comment and make it unsigned.

The value is intended for plugins to better correlate with Suricata's internal threads and is independent from the `slot_number` (which is used to form the per-thread EVE output filename).